src/js/routes/More/About.jsx

```diff
@@ -45,12 +45,12 @@ export default class About extends Component {
               Jenifer Fernandez Ancona - Oakland, CA<br />
               Rob Simpson - Warrenton, VA<br />
               Nicolas Fiorini - Arlington, VA<br />
+              Lisa Cho - San Francisco, CA<br />
```

Comment:
I think Lisa should be recognized above me and Nick for her hard work and major contributions